src/blog-post-workflow.js

```diff
@@ -293,7 +296,10 @@ const runWorkflow = async () => {
         }
       });
     }
-    // If ENABLE_SORT is false (i.e., disable_sort is true), postsArray remains in fetched order.
+    // If ENABLE_SORT is false (i.e., disable_sort is true), check if we need to reverse the feed order
+    else if (REVERSE_FEED_ORDER) {
```

Comment:
simply do `else if (SORT_ORDER === "desc" && !ENABLE_SORT)`

Reply:
Good call! I'll send the patch this evening.